### PR TITLE
Update README.md for 1.9

### PR DESCRIPTION
**Pull request by hrydgard**
1.9 is getting closer (matter of weeks now), so it's time again to list the new major stuff. Again, I'm sure I forgot many important things so let me know :) 

https://github.com/hrydgard/ppsspp/milestone/20

### CODE REVIEW

**Comment by LunaMoo** [comment]
Nothing big, but some QOL stuff to consider adding to the changelog:
````
* Added hotkeys for texture dump and replace (#11938)
* Added Visual Studio 2019 support finally saying bye bye to Windows XP dependency (#11995, others)
* Fixes for video dumping (#12069)
* Added a separate sound volume for alternative speed (#12124)
* Improved mouse control(Windows) (#12173, #12176)
* Support for installing texture packs from zips (#12175)
* Right analog support for touch controls(only used by patched games and some HD remasters) (#12182)
````

File: README.md [hunk 1]
```diff
@@ -23,6 +23,31 @@ For build instructions and other development tutorials, see [the wiki](https://g
...
+What's new in 1.9.0
+-------------------
+* Flicker fixed in God of War that appeared with newer drivers for Mali GPUs (#12224)
+* Improve performance of God of War on Vulkan (#12242), implement built-in GPU profiler (#12262, #12266)
+* Vertex range culling fixed on ARM Mali (#12227)
+* Started to improve VFPU precision, resulting so far in a fix for the long standing Tekken 6 leg shaking problem (#12217)
+* Fixed a VFPU precision snafu on ARM64, fixing disappearing officers in Warriors Orochi(#11299)
+* Vulkan is the default again on Android versions newer than Pie
+* Fix various homebrew store issues
+* GPU pause signal handling fixed, fixing some hangs (#12160)
```
[the remark this answers is not in the document]
Comment (hrydgard) [reply README.md]: Yeah, done.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,31 @@ For build instructions and other development tutorials, see [the wiki](https://g
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+What's new in 1.9.0
+-------------------
+* Flicker fixed in God of War that appeared with newer drivers for Mali GPUs (#12224)
+* Improve performance of God of War on Vulkan (#12242), implement built-in GPU profiler (#12262, #12266)
+* Vertex range culling fixed on ARM Mali (#12227)
+* Started to improve VFPU precision, resulting so far in a fix for the long standing Tekken 6 leg shaking problem (#12217)
+* Fixed a VFPU precision snafu on ARM64, fixing disappearing officers in Warriors Orochi(#11299)
```
[the remark this answers is not in the document]
Comment (hrydgard) [reply README.md]: Yup

**Comment by Emulatorer** [comment]
Maybe this two can be added to the v1.9 changelog
-Android OpenSL initial queue fix #12333
-PSP Camera Support (Android) #8521

**Comment by Emulatorer** [comment]
ppsspp version still not 1.9 in buildbot

**Comment by hrydgard** [comment]
That's correct, I haven't tagged the upgrade yet. Will do tonight.

**Comment by Emulatorer** [comment]
Ok I understand :)

**Pull request merged**